packages/frontend-2/components/viewer/anchored-point/Thread.vue

```diff
@@ -163,7 +163,7 @@
               </FormButton>
             </div>
             <div
-              v-if="!canReply && !isEmbedEnabled"
+              v-if="!activeUser && !isEmbedEnabled"
```

Comment:
Instead of removing the canReply check altogether, you should update the `useCheckViewerCommentingAccess()` composable to check that either: the user has a project role OR the project has `allowPublicComments` turned on.

The way you've configured it now is not correct, because you're allowing every logged in user to reply, even if the project doesn't have allowPublicComments turned on and the user does not have a project role.